examples/panel_kmeans.html

```diff
@@ -44,7 +44,6 @@
     <link rel="stylesheet" href="./assets/css/examples.css" />
     <link rel="stylesheet" href="./assets/prism/prism.css" />
     <script defer src="./assets/prism/prism.js"></script>
-    <script defer src="https://pyscript.net/latest/pyscript.js"></script>
```

Comment:
uhm... I wonder whether we could/should detect the "double include" case and emit a clean error instead of doing "random things"

Reply:
yeah I think that's a good idea, I was seeing some weird behaviour on the splash screen saying that the element was already created. I'll open an issue so I don't forget to tackle this 😄